### PR TITLE
signrpc: fix panic with missing UTXO information when signing Taproot output

### PR DESCRIPTION
**Pull request by guggero**
## Change Description
Fixes https://github.com/lightningnetwork/lnd/issues/6396.

Also fixes a value that wasn't copied to the RPC counterpart.

### CODE REVIEW

File: rpcserver.go [hunk 1]
```diff
@@ -2539,6 +2539,7 @@ func createRPCCloseUpdate(update interface{}) (
 			Update: &lnrpc.CloseStatusUpdate_ChanClose{
 				ChanClose: &lnrpc.ChannelCloseUpdate{
 					ClosingTxid: u.ClosingTxid,
+					Success:     u.Success,
```
Comment (bhandras) [inline rpcserver.go:2542]: :tada:

**Review by bhandras — APPROVED** [review]
LGTM :1st_place_medal:

**Review by Roasbeef — APPROVED** [review]
LGTM 🚦

**Comment by Roasbeef** [comment]
Needs a rebase then g2g!

**Pull request merged**